README.md

```diff
@@ -135,7 +141,9 @@ plotter.plot_classyfire_result(best_matches)
 Furthermore, Omigami provides the possibility to use the [NPClassifier](https://www.researchgate.net/publication/344008670_NPClassifier_A_Deep_Neural_Network-Based_Structural_Classification_Tool_for_Natural_Products) API.
 
 ```python
-from omigami import MoleculePlotter
+
+from omigami.plotting.comparison_plotter import MoleculePlotter
```

Comment:
you are importing this in omigami.__init__ as well. So I think `from omigami import MoleculePlotter` would also work?

Reply:
and same applies for above import

Reply:
then we don't need it in omigami.__init__ I think? or it won't hurt to have it there

Reply:
I will remove from omigami.init so the user has to import from plotting